docs/web3-eth.rst

```diff
@@ -419,8 +419,8 @@ getAccounts
 
     web3.eth.getAccounts([callback])
 
-Returns a list of accounts the node controls by using the provider and calling the RPC method ``eth_accounts``.
-If there are unlocked local accounts then it will return them instead of sending a request to the node.
+Will return a list of locally unlocked accounts or it will return the unlocked accounts from the currently connected node.
```

Comment:
For phrasing consistency, start with a verb. The difference between "locally [connected node]" "currently connected node" isn't clear.

Returns a list of the locally unlocked accounts, or if connected remotely, the exposed and unlocked accounts of the provider.